.github/workflows/issue-triage.yml

```diff
@@ -81,4 +81,31 @@ jobs:
             [`repl`](https://babeljs.io/repl/), or if the `repl` is
             insufficient, a new and minimal repository with instructions on
             how to build/replicate the issue.
-        
+
+  npm_cache:
+    name: Npm Cache
+    runs-on: ubuntu-latest
+    steps:
+      - name: Npm Cache Comment
+        uses: babel/actions/create-comment@v2
+        if: |
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'i: npm cache'
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          issue: ${{ github.event.issue.number }}
+          comment: >
+            Hi @${{ github.event.issue.user.login }}!
+
+            This seems to be an npm related issue.
+            Although we can't handle it, we provide some common workarounds for your reference.
+
+            1. Please check if there is a proxy or cache between you and npm
+            (this is the most likely cause and it is common in corporate networks).
+
+            2. Visit https://status.npmjs.org/ to see if the current npm is working properly.
+
+            3. Open the package's home page on npm like https://www.npmjs.com/package/@babel/core
+            and check if the version you want exists.
+
+            4. See https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67
```

Comment:
```suggestion
            This seems to be an npm related issue: you can verify that the package is available using the npm website.
            For example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types`
            (replace `@babel/types` in the URL with the package you want to check). If you see zero downloads don't worry:
            npm can take up to a few days to display the downloads count, but this does not mean that it's not working.
            
            Although we can't handle it, we provide some common workarounds for your reference.

            1. Please check if there is a stale proxy or cache between you and npm
            (this is the most likely cause and it is common in corporate networks).

            2. Visit https://status.npmjs.org/ to see if the current npm is working properly.

            3. See https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67
```

Reply:
Oops, I remembered to state the number of downloads, but I forgot it.😂